### ISSUE TITLE
Spongiosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#10935]
**Preferred term label:**
Spongiosis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Spongiosis refers to the increased intercellular edema in the epidermis resulting in the pulling away of the keratinocytes from their surrounding desmosomes.  Histologically, the spongiotic tissue reaction pattern involves the accumulation of fluid in the [intercellular spaces](https://www.sciencedirect.com/topics/medicine-and-dentistry/intercellular-space) of the epidermis, making the [desmosomes](https://www.sciencedirect.com/topics/medicine-and-dentistry/desmosome) junctions more easy to appreciate. Parakeratosis forms above the areas of spongiosis. The foci of spongiosis can be microscopic to grossly identifiable clinically with vesicles or bullae. PMID: 37445645; PMID: 28237387
Comment: Comment: Spongiosis is present in all diseases manifesting clinically as eczema; however, there are diseases with spongiosis that do not belong to the eczema spectrum, e.g., pityriasis rosea, erysipelas or tinea. . Spongiotic dermatoses include a very large list of disorders which are best grouped under neutrophilic, [eosinophilic](https://www.sciencedirect.com/topics/medicine-and-dentistry/eosinophilic), miliarial, follicular and pityriasiform forms of spongiosis.

**Parent term (use hpo.jax.org/app)**
Abnormal epidermal morphology HP:0011124

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Netherton syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
Adding new term: 
- ID: HP:6001166
- label: Spongiosis
- parents: [Abnormal epidermal morphology (HP:0011124)](https://hpo.jax.org/app/browse/term/HP:0011124)
- pmid: PMID:37445645|PMID:28237387
- def: Spongiosis refers to the increased intercellular edema in the epidermis resulting in the pulling away of the keratinocytes from their surrounding desmosomes. Histologically, the spongiotic tissue reaction pattern involves the accumulation of fluid in the intercellular spaces of the epidermis, making the desmosomes junctions more easy to appreciate. Parakeratosis forms above the areas of spongiosis. The foci of spongiosis can be microscopic to grossly identifiable clinically with vesicles or bullae.
- comment: Comment: Spongiosis is present in all diseases manifesting clinically as eczema; however, there are diseases with spongiosis that do not belong to the eczema spectrum, e.g., pityriasis rosea, erysipelas or tinea. . Spongiotic dermatoses include a very large list of disorders which are best grouped under neutrophilic, eosinophilic, miliarial, follicular and pityriasiform forms of spongiosis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10935
- synonyms:
